cala-ledger/src/tx_template/entity.rs

```diff
@@ -367,20 +367,20 @@ mod tests {
             .currency("'BTC'")
             .build()
             .unwrap()];
-        let new_journal = NewTxTemplate::builder()
+        let new_tx_template = NewTxTemplate::builder()
             .id(TxTemplateId::new())
             .code("CODE")
             .tx_input(
                 NewTxInput::builder()
-                    .effective("date('2022-11-01')")
+                    .effective("DATE('2022-11-01')")
```

Comment:
Why was this changed?

Reply:
was testing something. committed by mistake